transform/format.go

```diff
@@ -109,6 +122,30 @@ func (p *FormatParams) shortPath() (interface{}, error) {
 	return p.Info.Name(), nil
 }
 
+func (p *FormatParams) hash(hasher crypto.SignerOpts) (string, error) {
+	return hash(p.Info, p.Path, hasher)
+}
+
+func hash(info os.FileInfo, path string, hasher crypto.SignerOpts) (string, error) {
+	if info.IsDir() {
+		return "----------------------------------------", nil
```

Comment:
Should use [`string.Repeat`](https://golang.org/pkg/strings/#Repeat) with [`hasher.HashFunc().Size`](https://golang.org/pkg/crypto/#Hash.Size) here, since sizes may vary between hash functions.